hdr_html_canvas_element.md

```diff
@@ -112,77 +114,100 @@ Update `PredefinedColorSpace` to include the following new color spaces.
 
 #### `extended-srgb`
 
-To convert `extended-srgb` to XYZD50 under relative colorimetric intent, perform the following steps:
+* Transfer function: See [extended-linear-srgb](#extended-linear-srgb)
+
+* Connection matrix: matrix to convert the [sRGB primaries](https://www.w3.org/TR/css-color-4/#valdef-color-srgb) to the primary colours specified in Rec. ITU-R BT.2100
 
-* Convert each color channel to linear space.
-  * For each color channel value `x`, this means applying the following function:
-    * If `x < -0.4045`, then return `-pow((-x + 0.055)/1.055, 2.4)`
-    * Else if `x <= 0.4045` then return `x / 12.92`
-    * Else return `pow((x + 0.055)/1.055, 2.4)`
-* Apply the matrix transformation to convert the [sRGB primaries](https://www.w3.org/TR/css-color-4/#valdef-color-srgb) to the XYZ primaries.
-* Apply the matrix transformation to convert the [sRGB white point](https://www.w3.org/TR/css-color-4/#valdef-color-srgb) to the D50 white point.
+_Note:_ The domain of this transformation function is all real values. Its domain is not restricted to the unit interval [0, 1].
 
-Note that the domain of this transformation function is all real values. Its domain is not restricted to the unit interval [0, 1].
 Also see note in the Issues section at the bottom about whether this space should be distinct from the existing `'srgb'` space.
 
 #### `rec2100-hlg`
 
-To convert `rec2100-hlg` to XYZD50 under relative colorimetric intent, perform the following steps:
+* Transfer function: HLG Reference OETF specified at Rec. ITU-R BT.2100
 
-* Apply the HLG inverse OETF defined in Table 5 of [ITU-R 2100](https://www.itu.int/dms_pubrec/itu-r/rec/bt/R-REC-BT.2100-2-201807-I!!PDF-E.pdf).
-  * Note that this step of the transformation function is defined only on the domain of [0, 1].
-  * Pixel values outside of that domain are clamped to that domain.
-* Apply the matrix transformation to convert the primaries specified Table 2 of [ITU-R 2100](https://www.itu.int/dms_pubrec/itu-r/rec/bt/R-REC-BT.2100-2-201807-I!!PDF-E.pdf) to the XYZ primaries.
-* Apply the matrix transformation to convert the white point from the reference white specified Table 2 of [ITU-R 2100](https://www.itu.int/dms_pubrec/itu-r/rec/bt/R-REC-BT.2100-2-201807-I!!PDF-E.pdf) to the D50 white point.
+_Note:_ The range of the function is [0, 1].
 
-If this is followed, then converting from  `rec2100-hlg` to any SDR color space will not result in any luminance clipping.
-This is a desirable property.
+* Connection matrix: Identity
+
+_Note:_ Converting from `rec2100-hlg` to any SDR color space will not result in clipping.
 
 #### `rec2100-pq`
 
-To convert `rec2100-pq` to XYZD50 under relative colorimetric intent, perform the following steps:
+* Transfer function: `EOTF<sup>-1</sup>[F<sub>D</sub>/300]` where `EOTF<sup>-1</sup>` is the inverse of the Reference PQ EOTF specified at Rec. ITU-R BT.2100.
```

Comment:
If 300 is our final point if disagreement, we'll be in a very good place :D

Reply:
I think the gain will need to be parameterized as done in [TTML2](https://www.w3.org/TR/ttml2/#style-attribute-luminanceGain). I plan to file a separate issue.